server/nullauth/nullauth.go

```diff
@@ -9,7 +9,45 @@ import (
 	"github.com/buildbuddy-io/buildbuddy/server/util/status"
 )
 
-type NullAuthenticator struct{}
+func NewNullAuthenticator(anonymousUsageEnabled bool, adminGroupID string) *NullAuthenticator {
+	return &NullAuthenticator{
+		adminGroupID:           adminGroupID,
+		anonymousUsageDisabled: !anonymousUsageEnabled,
+	}
+}
+
+type NullAuthenticator struct {
+	adminGroupID           string
+	anonymousUsageDisabled bool
+}
+
+func (a *NullAuthenticator) AdminGroupID() string {
+	return a.adminGroupID
+}
+
+func (a *NullAuthenticator) AnonymousUsageEnabled() bool {
+	return !a.anonymousUsageDisabled
+}
+
+func (a *NullAuthenticator) EnableAnonymousUsage() {
```

Comment:
Are these setters (enable, disable, set) used anywhere?

Reply:
Not anymore, apparently; removed.
Thanks for the catch!